build-manual

```diff
@@ -11,6 +11,6 @@ ESCAPED_REQS=$(awk '{printf "%s%s",sep,$0; sep="\\\n"} END{print ""}' ../manual_
 sed "s/REQUIREMENTS/$ESCAPED_REQS/" index-rst-template > index.rst
 
 rm -rf build/
-sphinx-build -b html . build -W
```

Comment:
Why remove this?

Reply:
There is a warning when building locally from pytket.zx. This wasn't showing up on CI as the build didn't rerun on pushes as I expected.

Reply:
I think we can just add `python -m pip install kahypar` to the `check-manual` workflow, and this warning will go away (so that we can restore the `-W`).

Reply:
Thanks, works for me locally as well